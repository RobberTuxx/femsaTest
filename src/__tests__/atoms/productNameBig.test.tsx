import React from 'react';
import renderer from 'react-test-renderer';
import ProductNameBig from "../../components/atoms/productNameBig";
it('render product name big', ()=> {
    const component = renderer.create(<ProductNameBig text={"Hello world!"}/>).toJSON()
    expect(component).toMatchSnapshot()
})
it('has 0 child', ()=> {
    const component = renderer.create(<ProductNameBig text={"Hello world!"}/>).toJSON()
    expect(component.children.length).toBe(1)
})
