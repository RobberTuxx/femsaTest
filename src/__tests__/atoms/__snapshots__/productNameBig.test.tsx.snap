// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render product name big 1`] = `
<Text
  style={
    {
      "color": "#000000",
      "fontFamily": "Avenir",
      "fontSize": 37.5,
      "fontWeight": "800",
    }
  }
>
  Hello world!
</Text>
`;
